bin/aeproject/cli-commands/init/artifacts/exampleTests.js

```diff
@@ -14,33 +14,33 @@
  *  OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
  *  PERFORMANCE OF THIS SOFTWARE.
  */
-const Ae = require('@aeternity/aepp-sdk').Cli;
+const Ae = require('@aeternity/aepp-sdk').Universal;
 
 const config = {
-	host: "http://localhost:3001/",
-	internalHost: "http://localhost:3001/internal/",
-	ownerKeyPair: {
-		secretKey: 'bb9f0b01c8c9553cfbaf7ef81a50f977b1326801ebf7294d1c2cbccdedf27476e9bbf604e611b5460a3b3999e9771b6f60417d73ce7c5519e12f7e127a1225ca',
-		publicKey: 'ak_2mwRmUeYmfuW93ti9HMSUJzCk1EYcQEfikVSzgo6k2VghsWhgU'
-	},
-	pubKeyHex: '0xe9bbf604e611b5460a3b3999e9771b6f60417d73ce7c5519e12f7e127a1225ca',
+  host: "http://localhost:3001/",
+  internalHost: "http://localhost:3001/internal/",
```

Comment:
Please use this:
url : `http://localhost:3013`,
internalUrl : `http://localhost:3113`